src/nimdowpkg/config/tagsettings.nim

```diff
@@ -31,19 +32,30 @@ proc parseTagSetting(tagSetting: var TagSetting, settingsTable: TomlTableRef) =
       raise newException(Exception, "Invalid numMasterWindows for tag")
     tagSetting.numMasterWindows = numMasterWindows.intVal.int
 
+  if settingsTable.hasKey("defaultMasterWidthPercentage"):
+    let masterWidthSetting = settingsTable["defaultMasterWidthPercentage"]
+    if masterWidthSetting.kind == TomlValueKind.Int:
+      tagSetting.defaultMasterWidthPercentage = masterWidthSetting.intVal.int.clamp(10,90)
```

Comment:
I think here we should check if they provided a value outside of the 10-90 range, as well. 
Something like:
```nim
let givenPercent = masterWidthSetting.intVal
if givenPercent < 10 or givenPercent > 90:
  log "Invalid defaultMasterWidthPercentage, clamped to 10-90%", lvlWarn
tagSetting.defaultMasterWidthPercentage = givenPercent.clamp(10, 90)
```